snek/exts/randomization.py

```diff
@@ -0,0 +1,164 @@
+from random import choice, randint, random
+from typing import Callable, Tuple, Union
+
+from discord import Embed, Message
+from discord.ext.commands import BadArgument, Cog, Context, MessageConverter, command
+
+from snek.bot import Snek
+
+UWU = {
+    "fi": "fwi",
+    "l": "w",
+    "r": "w",
+    "some": "sum",
+    "th": "d",
+    "thing": "fing",
+    "tho": "fo",
+    "you're": "yuw'we",
+    "your": "yur",
+    "you": "yuw",
+}
+
+MAGICBALL = {
+    'Affirmative': ['It is certain', 'It is decidedly so.', 'Without a doubt.',
+                    'Yes - definitely.', 'You may rely on it', 'As I see it, yes.',
+                    'Most likely', 'Outlook good.', 'Yes.', 'Signs point to yes.'],
+    'Non-committal': ['Reply hazy, try again.', 'Ask again later.', 'Better not tell you now.',
+                      'Cannot predict now.', 'Concentrate and ask again.'],
+    'Negative': ["Don't count on it.", 'My reply is no.', 'My sources say no.',
+                 'Outlook not so good.', 'Very doubtful.']
+}
+
+
+class Randomization(Cog):
+    """Randomizing the contents of messages."""
+
+    def __init__(self, bot: Snek) -> None:
+        self.bot = bot
+
+    @command(name='uwu')
+    async def uwu_case(self, ctx: Context, *, msg: str) -> None:
+        def func(string: str, stutter_rate: float = 0.1) -> str:
+            def _stutter(word: str) -> str:
+                while random() < stutter_rate:
+                    word = word[0] + '-' + word
+
+                return word
+
+            # UWU the string
+            for before, after in UWU.items():
+                string = string.replace(before, after)
+
+            # Add stutter
+            return ' '.join(map(_stutter, string.split(' ')))
+
+        msg, embed = await Randomization._get_text_and_embed(ctx, msg)
```

Comment:
I think it's better practice to just use `self` here, and similarly in other places where you call static methods.
```suggestion
        msg, embed = await self._get_text_and_embed(ctx, msg)
```